src/register/RegistrationPage.jsx

```diff
@@ -683,7 +679,7 @@ class RegistrationPage extends React.Component {
                 { this.addExtraRequiredFields() }
                 <ValidationFormGroup
                   for="optional"
-                  className="custom-control pt-10 mb-0"
```

Comment:
This shouldn’t be removed. We need zero bottom margin on this.

Reply:
done